community/kernel/src/test/java/org/neo4j/kernel/impl/api/state/LabelTransactionStateTest.java

```diff
@@ -362,23 +356,21 @@ private void commitLabels( Integer... labels ) throws Exception
 
     private void assertLabels( Integer... labels ) throws EntityNotFoundException
     {
-        try ( Cursor<NodeItem> cursor = txContext.nodeCursorById( state, nodeId ) )
-        {
-            if ( cursor.next() )
-            {
-                assertEquals( asSet( labels ), PrimitiveIntCollections.toSet( cursor.get().getLabels() ) );
-            }
-        }
+        txContext.nodeCursorById( state, nodeId ).forAll( node -> assertEquals( asSet( labels ),
+                node.labels().mapReduce( new HashSet<>(), IntSupplier::getAsInt, this::addToCollection ) ) );
```

Comment:
Yeah maybe a `collect` makes sense.

We can use a primitive collection but I didn't think it was important in a test.